src/platform/testing/contract/index.js

```diff
@@ -0,0 +1,29 @@
+import { Pact } from '@pact-foundation/pact';
+
+/**
+ * Helper for writing a Pact contract test that abstracts the boilerplate
+ * for setting up the mock API and using it to verify the mocked interactions.
+ *
+ * @param {string} consumer - String label for the consumer.
+ * @param {string} provider - String label for the provider.
+ * @param {function} test - Callback function for a test suite (describe block).
+ */
+const contractTest = (consumer, provider, test) => {
+  describe(consumer, () => {
+    const mockApi = new Pact({
+      port: 3000,
+      consumer,
+      provider,
+      spec: 2,
```

Comment:
Is this specifying which version of the Pact spec is being used? 

https://docs.pact.io/getting_started/terminology/#pact-specification

Reply:
Yes, that's correct. [The full list of params are under "Constructor" here](https://github.com/pact-foundation/pact-js#api).